src/Startup/Backend/Monitor.vala

```diff
@@ -49,14 +49,16 @@ public class Startup.Backend.Monitor : Object, Port.Monitor {
     void on_change_occurred (File file, File? dest, FileMonitorEvent event) {
         var path = file.get_path ();
 
-        if (Utils.is_desktop_file (path) == false)
+        if (Utils.is_desktop_file (path) == false) {
             return;
+        }
 
-        if (event == FileMonitorEvent.CREATED)
+        if (event == FileMonitorEvent.CREATED) {
             file_created (path);
-        else if (event == FileMonitorEvent.DELETED)
+        } else if (event == FileMonitorEvent.DELETED) {
             file_deleted (path);
-        else if (event == FileMonitorEvent.CHANGED)
+        } else if (event == FileMonitorEvent.CHANGED) {
```

Comment:
If we're modifying this we might as well make it a switch statement instead of else/ifs